var/spack/repos/builtin/packages/netlib-scalapack/package.py

```diff
@@ -39,6 +39,8 @@ class NetlibScalapack(CMakePackage):
     depends_on('blas')
     depends_on('cmake', when='@2.0.0:', type='build')
 
+    patch("cmake_fortran_mangle.patch", when='@2.0.2:')
```

Comment:
@luszczek Could you add a comment which refers to the link where you submitted the patch: e.g.

```
# See: https://github.com/Reference-ScaLAPACK/scalapack/issues/9
```

Also, should this apply to exactly version `2.0.2`, i.e. `when='@2.0.2')` since you mention they merged this?